windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -75,55 +75,3 @@ Windows nodes are now fully integrated with most of the monitoring capabilities
 * The filesystem graphs available in the web console do not display for Windows nodes. This is caused by changes in the filesystem queries. This will be fixed in a future release of WMCO. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1930347[*BZ#1930347*])
 
 * The Prometheus windows_exporter used by the WMCO currently collects metrics through HTTP, so it is considered unsafe. You must ensure that only trusted users can retrieve metrics from the endpoint. The windows_exporter feature recently added support for HTTPS configuration, but this configuration has not been implemented for WMCO. Support for HTTPS configuration in the WMCO will be added in a future release.
-
-* If you have a cluster with two Windows nodes, and you create a web server deployment with two replicas, each pod lands on a Windows compute node. In this scenario, if you create a `Service` object with type `LoadBalancer`, communication with the load balancer endpoint is not reliable. To mitigate this issue for clusters installed on AWS or Azure, you must use Windows Server 2019 version 10.0.17763.1457 or earlier. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1905950[*BZ#1905950*])
```

Comment:
Unfortunately customers are going to hit [Bug 1939968](https://bugzilla.redhat.com/show_bug.cgi?id=1939968) - `kube-proxy service terminated unexpectedly after recreated LB service now`. So we might need to call that out. Though there is a hack to work around this:
> if you don't wait for the deployment pods to go to running and create the LB service immediately, kube-proxy does not crash on the nodes.

Reply:
@aravindhp I just added a new known issue for [BZ#1939968](https://bugzilla.redhat.com/show_bug.cgi?id=1939968) in the latest commit. Would that provide adequate coverage of our current state?

Reply:
LGTM